TypeCobol/AntlrGrammar/Cobol2002CodeElements.g4

```diff
@@ -75,3 +77,14 @@ excludedDataItem:
 
 jsonStatementEnd:
 	END_JSON;
+
+// New Cobol v6.2 JSON PARSE statement. Converts JSON text to COBOL data formats.
+jsonParseStatement:
+	JSON ({ string.Equals(CurrentToken.Text, "PARSE", System.StringComparison.OrdinalIgnoreCase) }? KeywordPARSE=UserDefinedWord source=storageArea1)
+	INTO destination=variable1
+	(WITH? DETAIL)? 
+	(name OF? jsonParseNameMapping+)? // Re-use of contextual keyword NAME defined for XML GENERATE in CobolCodeElements.
+	(SUPPRESS excludedDataItem+)?;
+
+jsonParseNameMapping:
+	dataItem=variable1 IS? (OMITTED|outputName=alphanumericValue2);
```

Comment:
I assume you copied the naming from JSON GENERATE here so it's ok but the role of items in the name mapping are reversed so the name 'outputName' is misleading. This is actually an inputName because it refers to a name in the input string to be parsed.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1542/commits/70bde4b647a278978958313a518d6f507e1083c5